enterprise/frontend/src/metabase-enterprise/content_verification/metrics.tsx

```diff
@@ -0,0 +1,83 @@
+import { type ChangeEvent, useCallback } from "react";
+import { t } from "ttag";
+
+import type {
+  MetricFilterControlsProps,
+  MetricFilterSettings,
+} from "metabase/browse/utils";
+import { useUserSetting } from "metabase/common/hooks";
+import { getSetting } from "metabase/selectors/settings";
+import { Button, Icon, Paper, Popover, Switch, Text } from "metabase/ui";
+import type { State } from "metabase-types/store";
+
+const USER_SETTING_KEY = "browse-filter-only-verified-metrics";
+
+export function getDefaultMetricFilters(state: State): MetricFilterSettings {
+  return {
+    verified: getSetting(state, USER_SETTING_KEY) ?? false,
+  };
+}
+
+export const MetricFilterControls = ({
```

Comment:
You could add a comment here noting that this component is very similar to `ModelFilterControls`. I don't think it's necessary just yet to unify the two components, but it might eventually become helpful to do that – for example, if a similar filtering mechanism is needed for a third kind of entity besides models and metrics. A comment like that might help a future dev reason about this unification.